soc/arm/nordic_nrf/soc_irq.c

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <sw_isr_table.h>
+
+#ifdef CONFIG_DYNAMIC_INTERRUPTS
+unsigned int _irq_line_get(void)
+{
+	unsigned int irqn;
+
+	__asm__ volatile("mrs %0, IPSR\n\t" : "=r"(irqn));
```

Comment:
Use, here CMSIS C function

Reply:
Where? I couldn't find it.

Reply:
See how the z_arch_is_in_isr() is implemented ;)

Reply:
Thank you. :)